nearbridge/NearBridge.full.sol

```diff
@@ -370,6 +370,7 @@ library Borsh {
     function decodeBytes32(Data memory data) internal pure shift(data, 32) returns(byte[32] memory value) {
         bytes memory raw = data.raw;
         uint256 offset = data.offset;
+        // solium-disable-next-line security/no-inline-assembly
```

Comment:
Add these changes to the original file in `contracts` directory and produce this full file with `yarn run dist` or `npm run dist`.

Reply:
These changes were autogenerated by `yarn run dist`